drivers/vcluster/vcluster.go

```diff
@@ -0,0 +1,97 @@
+package vcluster
+
+import (
+	"fmt"
+	"github.com/portworx/torpedo/pkg/log"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+var (
+	UpdatedClusterContext string
+	CurrentClusterContext string
+	ContextChange         = false
+)
+
+// This method switches kube context between host and any vcluster
+func SwitchKubeContext(target string) error {
+	cmd := exec.Command("kubectl", "config", "get-contexts", "-o", "name")
```

Comment:
Hi @nikolaypopov , thanks for this suggestion. I was also exploring this on similar lines. The problem here is - there is no direct API endpoint exposed for kubectl config cli, as this works directly with kubeconfig files. So from endpoint perspective, we need to provide path to kubeconfig and then the API would read that file and use that config. However, here in vcluster context, kubeconfig is not static as these vclusters are ephemeral in nature and our test cases will be creating and destroying them on need basis. We can only get kubeconfig out of a vcluster if we have a separate IP/hostname for that vcluster and through load-balancer we are trying to connect over it. Since we are not going that route, and simply using port-forwarding mechanism of vclusters to connect we won't get kubeconfigs here. 

This is what I researched and then decided to go ahead with this way. Please let me know if you are able to figure out any other way for this usecase. I can implement that otherwise I believe this is the only way as of now.